src/model/model.ts

```diff
@@ -240,7 +252,22 @@ export class Model {
       }
     }
 
-    ast.policy = ast.policy.concat(rules);
+    let priorityFlag = false;
+
+    if (rules.length > 0) {
+      const tmpRule = rules[0];
+      if (tmpRule.find((value) => value === 'p_priority')) {
+        priorityFlag = true;
```

Comment:
incorrect implementation.

correct implementation in https://github.com/casbin/node-casbin/pull/250/files#diff-58a5f4a5424e479db505f909fa639aea8fbb996324bc73430934932455134dffR227.

Reply:
My fault, I messed up it.